src/components/Confirm/index.js

```diff
@@ -164,7 +164,6 @@ class Confirm extends Component  {
     else {
       this.confirmAndProceed(apiResponse)
     }
-    this.setState({uploadInProgress: false})
```

Comment:
Removing this line of code won't allow you to display the confirm screen with warn if glare is detected, it will stay on the spinner. This is also the reason why cumber tests are failing.